src/test/runTest.ts

```diff
@@ -24,12 +24,17 @@ async function main() {
       resolveCliPathFromVSCodeExecutablePath(vscodeExecutablePath);
 
     // Install extension dependencies
-    extensionDependencies.forEach((dependency) => {
-      cp.spawnSync(cliPath, ["--install-extension", dependency], {
+    cp.spawnSync(
```

Comment:
Installing the extensions one at a time was flaky for some reason.  Installing them with one command seems better anyways